docs/content/integrations/dbt/reference.mdx

```diff
@@ -131,62 +133,179 @@ Refer to the [Schedule documentation](/concepts/partitions-schedules-sensors/sch
 
 ---
 
-## Understanding asset keys
+## Understanding asset definition attributes generated for dbt nodes
 
-In Dagster, each asset has an asset key to identify it. Dagster automatically generates these keys for each dbt node in the project as well as the sources for each node.
+In Dagster, each asset has definition attributes. Dagster automatically generates these attributes for each dbt node in the project that is represented as a software-defined asset. These attributes can optionally be overridden by the user.
```

Comment:
perhaps "each asset definition has attributes", and "Dagster automatically generates these attributes for each software-defined asset loaded from the dbt project".